app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user&.authenticate params[:session][:password]
```

Comment:
`&.` có tác dụng gì em

Reply:
- Safe Navigation Operator (&.):
- value_check&.do_some_thing: return nil when value_check is nil (trường hợp còn lại => raise exception)